src/Compilers/Test/Core/Assert/ClrOnlyFactAttribute.cs

```diff
@@ -79,15 +77,4 @@ private static string GetSkipReason(ClrOnlyReason reason)
             }
         }
     }
-
-    public sealed class MonoOnlyFactAttribute : FactAttribute
-    {
-        public MonoOnlyFactAttribute(string reason)
-        {
-            if (!MonoHelpers.IsRunningOnMono())
```

Comment:
I imagine there are still other usages of this helper?

Reply:
Yeah there are multiple usages.